upstream/version.go

```diff
@@ -70,6 +70,15 @@ func forURL(url string) (Version, error) {
 		if len(match) > 0 {
 			return rubygem(match[1]).latestVersion()
 		}
+	// A fallback for if gitlab isn't in the domain name
+	case strings.Contains(url, "/-/archive/"):
```

Comment:
This fallback might result in many false-positive matches? Are you aware of an instance requiring this match? I'd rather remove this and explicitly add needed domains.

Reply:
I wasn't so sure whether it was needed either, as I only know of the Gnome public Gitlab instance, which contains `gitlab` in the domain name. I merely added it because of the unknown possibility of someone running a self-hosted instance without Gitlab in the domain name.

I guess it wouldn't hurt if it was removed, but then individual matches will have to be added for anyone that doesn't use `gitlab` in their domain.